src/resources/js/front-page-setting.js

```diff
@@ -0,0 +1,27 @@
+var tribe_events_front_page_setting = tribe_events_front_page_setting || {};
```

Comment:
Because you can apply the 'use strict' to the whole file without affecting anything else. And normally with this you prevent 1 script breaking the whole page.